### PR TITLE
ppx_protocol_conv 2.0.0

### PR DESCRIPTION
**Pull request by andersfugmann**

### CODE REVIEW

**Comment by hcarty** [comment]
@avsm It looks like at least some of the failures are related to an issue with yaml.  Is this related to the recent 0.11 Jane Street releases?

```
#=== ERROR while installing yaml.0.1.0 ========================================#
# opam-version         1.2.2 (aa258ecc06d3aea5a67f442a4ffd23f2a457180b)
# os                   linux
# command              jbuilder build -p yaml -j 2
# path                 /home/opam/.opam/4.05.0/build/yaml.0.1.0
# compiler             4.05.0
# exit-code            1
# env-file             /home/opam/.opam/4.05.0/build/yaml.0.1.0/yaml-7-674208.env
# stdout-file          /home/opam/.opam/4.05.0/build/yaml.0.1.0/yaml-7-674208.out
# stderr-file          /home/opam/.opam/4.05.0/build/yaml.0.1.0/yaml-7-674208.err
### stderr ###
# [...]
# Error: Unbound module Sexplib
#           cc vendor/scanner.o
# scanner.c: In function 'yaml_parser_decrease_flow_level':
# scanner.c:1189:23: warning: variable 'dummy_key' set but not used [-Wunused-but-set-variable]
#      yaml_simple_key_t dummy_key;    /* Used to eliminate a compiler warning. */
#                        ^~~~~~~~~
#     ocamlopt types/bindings/.yaml_bindings_types.objs/yaml_bindings_types.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.05.0/bin/ocamlopt.opt -w -40 -g -I types/bindings/.yaml_bindings_types.objs -I /home/opam/.opam/4.05.0/lib/base -I /home/opam/.opam/4.05.0/lib/base/caml -I /home/opam/.opam/4.05.0/lib/base/shadow_stdlib -I /home/opam/.opam/4.05.0/lib/bytes -I /home/opam/.opam/4.05.0/lib/ctypes -I /home/opam/.opam/4.05.0/lib/integers -I /home/opam/.opam/4.05.0/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.05.0/lib/sexplib0 -no-alias-deps -o types/bindings/.yaml_bindings_types.objs/yaml_bindings_types.cmx -c -impl types/bindings/yaml_bindings_types.pp.ml)
# File "types/bindings/yaml_bindings_types.ml", line 15, characters 5-17:
# Error: Unbound module Sexplib
```

File: packages/ppx_protocol_conv/ppx_protocol_conv.2.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Anders Fugmann <anders@fugmann.net>"
+authors: "Anders Fugmann"
+homepage: "https://github.com/andersfugmann/ppx-deriving-protocol"
```
Comment (hcarty) [inline packages/ppx_protocol_conv/ppx_protocol_conv.2.0.0/opam]: @andersfugmann It looks like these links all point to a deprecated repository.

File: packages/ppx_protocol_conv/ppx_protocol_conv.2.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+dev-repo: "git+https://github.com/andersfugmann/ppx-deriving-protocol"
+bug-reports: "https://github.com/andersfugmann/ppx-deriving-protocol/issues"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+build-test: [["jbuilder" "runtest" "-j" jobs]]
+version: "2.0.0"
```
Comment (hcarty) [inline packages/ppx_protocol_conv/ppx_protocol_conv.2.0.0/opam]: The `version` line can be removed from the `*.opam` files.  It is inferred based on the repository's directory structure.

File: packages/ppx_protocol_conv/ppx_protocol_conv.2.0.0/opam (hunk 1 above)
Comment (andersfugmann) [reply packages/ppx_protocol_conv/ppx_protocol_conv.2.0.0/opam]: Thanks. Nice catch.

File: packages/ppx_protocol_conv/ppx_protocol_conv.2.0.0/opam (hunk 2 above)
Comment (andersfugmann) [reply packages/ppx_protocol_conv/ppx_protocol_conv.2.0.0/opam]: Ill remove them.

**Comment by hcarty** [comment]
Merging.  Thanks for the submission and sorry for the delay!

**Comment by andersfugmann** [comment]
Thanks

**Pull request merged**